package-lock.json

```diff
@@ -121,6 +121,14 @@
       "dev": true,
       "requires": {
         "semver": "5.6.0"
+      },
```

Comment:
@manast This is another example of why I think `package-lock.json` should not be checked in. Unless I'm mistaken, there's no reason it should be updated as part of this PR, and yet `npm` loves to make changes to it.

Reply:
yes, for libraries it should not be checked-in, I will remove it.

Reply:
Sounds good. Since the lib ended up getting removed anyway, I can revert the change and commit that with my next commit so that `package-lock.json` remains unchanged for this PR.